tests/try/basic/data/epel.exp

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/expect -f
+# Try epel
+
+set timeout 60
```

Comment:
I set timeout to 60 seconds as default timeout is 10 seconds and the provision to virtual machine is taking more than 10 seconds.

Reply:
Increased timeout to 180 seconds taking into consideration boot timeout (2m) and assuming the image is pre-fetched (if the image isn't pre-fetched we should make it work)